content/articles/where-incorporate-eu-citizen.md

```diff
@@ -0,0 +1,157 @@
+Incorporating is very simple and usually only takes a day or so. Most accountants will offer incorporation as a freebie. 
+You can open a bank account remotely and most banks in the UK (at least the one I've seen) offer a good online banking experience.
+
+With an accountant and a tool like FreeAgent or Xero, you don't really need to do any significant business administration work. We haven't run into issues with HMRC (the revenue service in the UK) that you are dealing with as a company but I have heard many horror stories about it.
+
+Virtual offices are easy to find in London so you don't have to live in the country.
+
+### Ireland
+
+#### Trivia
+The other obvious choice when it comes to English speaking countries in the EU. It is also famous for being the main place for big companies to do tax avoidance with some special arrangement called the [double irish arrangement](https://en.wikipedia.org/wiki/Double_Irish_arrangement).
+
+Ireland is very business friendly, doesn't seem to want to ban encryption any time soon and its culture is similar to the one in the UK.
+
+Its politics seem very driven by religion and they have actually introduced a [blasphemy law](https://en.wikipedia.org/wiki/Blasphemy_law_in_the_Republic_of_Ireland) recently (2009), which seems hard to believe coming.
+
+#### Taxes
+The corporation tax is **12.5%** and it might get interesting if the UK and Ireland get into a lower corporation tax war. The dividend tax rate is a flat **20%**.
+
+There is also a **tax exemption for the first three years** if the company's corporation tax to pay does not exceed €40,000, which corresponds to €320,000 of profit at the 12.5% tax rate. In short, unless you're growing super fast in terms of profit, you're not paying any corporation tax initially.
+
+#### How to incorporate/open bank account
+Incorporating in Ireland takes 2-3 days and cost €320 euros. Opening a bank account should be possible remotely but it might be easier to do a day trip to Dublin.
+
+Virtual offices are easy to find in Dublin so you don't have to live in the country.
+
+### Estonia
+
+#### Trivia
+I'll be the first one to say I don't know much about Estonia. Apparently most of the interactions with the government and banks can be done in english but having a local partner is useful for the more complex cases where it might be needed. Since you should get a local accountant, this shouldn't be an issue.
+
+The political environment seems pretty quiet and the current leading political party, the [Estonia Reform Party](https://en.wikipedia.org/wiki/Estonian_Reform_Party) is advocating for even more liberalism when it comes to companies.
+
+Estonia ranks 1st on the [Barclays Digital Development index](https://digitalindex.uk.barclays/ranking) and 16th on the [Doing Business ranking](http://www.doingbusiness.org/rankings)
+
+Do note that Stripe is not available in Estonia yet so it might be a deal breaker if your business depends on it.
```

Comment:
Might be worth tweeting stripe to ask about estimations. They may become available sooner than expected